ci/run.bat

```diff
@@ -1,7 +1,8 @@
-call dub test --skip-registry=all --compiler=%DC%
-::if %errorlevel% neq 0 exit /b %errorlevel%
-call rdmd --compiler=%DC% ./tests/runner.d --compiler=%DC% -cov
-::if %errorlevel% neq 0 exit /b %errorlevel%
+rem call dub test --skip-registry=all --compiler=%DC%
+call dub build --skip-registry=all --compiler=%DC% -c unittest -b unittest
```

Comment:
```suggestion
rem We are currently only building agora in unittest module, but not actually running it
rem because it crashes. When fixed, the following line can be uncommented:
rem call dub test --skip-registry=all --compiler=%DC%
call dub build --skip-registry=all --compiler=%DC% -c unittest -b unittest
```

If I understand correctly what's happening here ? But before we were running the tests, what happened ?

Reply:
before
‘dub build --skip-registry=all --compiler=%DC% -c unittest -b unittest’

Reply:
https://github.com/bpfkorea/agora/pull/808#issuecomment-628411471
Another error in the unit test in Windows is solved at #820

Reply:
Applied the suggestion.